tests/server_test.py

```diff
@@ -60,16 +60,15 @@ def test_simulation_redirect(fc):
 
 def test_simulation_schema(fc):
     from pykern.pkcollections import PKDict
-    from pykern.pkunit import pkexcept
+    from pykern.pkunit import pkexcept, pkok
 
     r = fc.sr_post_form(
         "simulationSchema",
         data=PKDict(simulationType="myapp"),
     )
-    for k in ("model", "view"):
-        assert k in r.keys()
-    for k in ("dog", "heightWeightReport"):
-        assert k in r.model.keys()
+    pkok(isinstance(r.model, PKDict), "expected model to be PKDict")
```

Comment:
This check is sufficiently tested by the r.model.dog check. It's better to not test too much stuff. If something comes back with r.model.dog as an attribute, then it's likely a schema. It could be data so you could go one level deeper, but that's not likely to be broken. Tests should be as minimal as possible to accomplish the goal.

Reply:
[e474deb](https://github.com/radiasoft/sirepo/pull/6921/commits/e474debba2686e7e319531ca53708db538f023f7)